lib/postmark/message_extensions/mail.rb

```diff
@@ -15,6 +15,16 @@ def tag=(val)
       header['TAG'] = val
     end
 
+    # @val accepts :html_only, :text_only, :html_and_text, :none
```

Comment:
I’m not sure who this comment is for :) If you want to communicate some restriction to the user, it’s better to throw an `ArgumentError` if a given value isn’t supported, e.g.:

``` ruby
LINK_TRACKING_MODES = %w(HtmlOnly TextOnly HtmlAndText None)

def track_links=(val)
  mode = ::Postmark::Inflector.to_postmark(val) # Doing this first allows passing the mode in Postmark format
  raise ArgumentError, "#{mode} is not a supported link tracking mode. Valid modes are #{LINK_TRACKING_MODES.join(', ')}" unless LINK_TRACKING_MODES.include?(mode)
  header['TRACK-LINKS'] = mode
end
```

I have to note that this approach has the drawback of having to update the gem each time we add a new link tracking mode. If there is already validation on the API, I’d *not* add any restriction and remove the comment, too. You can then list currently supported modes in the README (or a dedicated Wiki page).